src/main/java/net/minecraftforge/client/model/pipeline/BlockInfo.java

```diff
@@ -114,16 +113,12 @@ public void updateLightMatrix()
                     s[x][y][z] = (brightness >> 0x14) & 0xF;
                     b[x][y][z] = (brightness >> 0x04) & 0xF;
                     ao[x][y][z] = state.getAmbientOcclusionLightValue();
-                    if(x == 1 && y == 1 && z == 1)
-                    {
-                        full = state.isFullCube();
-                    }
                 }
             }
         }
-        if(!full)
+        for(EnumFacing side : EnumFacing.values())
         {
-            for(EnumFacing side : EnumFacing.values())
+            if(!state.getBlock().doesSideBlockRendering(state, world, blockPos, side))
```

Comment:
If you're doing this here, shouldn't you do an analogous change in VertexLighterFlat.updateLightmap?